stockflux-launcher/src/App.js

```diff
@@ -1,25 +1,36 @@
-import React from 'react';
+import React, {useRef, useEffect} from 'react';
 import cx from 'classnames';
 import 'stockflux-components';
 import { WindowHooks, Constants } from 'openfin-react-hooks';
 import {
   FaChevronUp,
   FaChevronLeft,
   FaChevronRight,
+  FaRegHandRock
 } from 'react-icons/fa';
 import AppShortcuts from './app-shortcuts/AppShortcuts';
 import FreeTextSearch from './free-text-search/FreeTextSearch';
 import ToolBar from './toolbar/ToolBar';
 import CloseButton from './toolbar/CloseButton';
 import './App.css';
 
+const isLauncherHorizontal = (edge) => {
+  return edge === Constants.ScreenEdge.TOP;
+}
+
 export default () => {
   const [edge, windowActions] = WindowHooks.useDockWindow(Constants.ScreenEdge.TOP, window.fin.Window.getCurrentSync(),
       true, { dockedWidth: 50, dockedHeight: 50 });
 
+  const prevEdgeRef = useRef();
+      useEffect(() => {
+        prevEdgeRef.current = edge;
+  });
+  const prevEdge = prevEdgeRef.current;    
```

Comment:
Could instead define the edge to check here, saving a few repeated conditions
e.g. `const edgeToCheck = edge === Constants.ScreenEdge.NONE ? prevEdgeRef.current : edge `